docs/posts/ibis-version-9.0.0-release/index.qmd

```diff
@@ -0,0 +1,351 @@
+---
+title: "Ibis 9.0: SQLGlot-ification"
+author: "Ibis team"
+date: "2024-04-18"
+categories:
+    - release
+    - blog
+---
+
+## Overview
+
+Ibis 9.0 completes ["the big
```

Comment:
```suggestion
Ibis 9.0 wraps up ["the big
```
Maybe? It just seems repetitive to say "completes" and then "completing" right after.